docs/pages/deployment/docker.rst

```diff
@@ -0,0 +1,131 @@
+.. _running-docker:
+
+Running on Docker
+#################
+
+This guide helps you to configure the Nuts node in Docker.
+To use the most recent release use ``nutsfoundation/nuts-node:latest``. For production environments it's advised to use a specific version.
+
+Examples
+********
+The examples assume the current working directory on the host contains the following structure:
+
+.. code-block:: none
+
+    .
+    ├── config
+    │   └── nuts.yaml
+    └── data
+
+The ``data`` directory must be owned by container user ``18081``. (``chown -R 18081:18081 ./data``)
+
+These examples use the following ``nuts.yaml`` configuration file:
+
+.. code-block:: yaml
+
+  strictmode: false
+  http:
+    internal:
+      address: :8081
+
+See the :ref:`configuration<nuts-node-config>` documentation how to setup the ``nuts.yaml`` to meet your needs.
+
+Docker Compose
+^^^^^^^^^^^^^^
+
+Copy the following YAML file and save it as ``docker-compose.yaml`` in the working directory.
+
+.. code-block:: yaml
+
+  services:
+    nuts:
+      image: nutsfoundation/nuts-node:latest
+      ports:
+        - 8080:8080
+        - 8081:8081
+      volumes:
+        - "./nuts.yaml:/nuts/nuts.yaml:ro"
+        - "./data:/nuts/data:rw"
+
+Start the service:
+
+.. code-block:: shell
+
+  docker compose up
+
+Docker ``run``
+^^^^^^^^^^^^^^
+
+If you want to run without Docker Compose you can use the following command from the working directory:
+
+.. code-block:: shell
+
+  docker run --name nuts -p 8080:8080 -p 8081:8081 \
+    --mount type=bind,source="$(pwd)",target=/nuts \
+    nutsfoundation/nuts-node:latest
+
+.. note::
+
+    The command above uses ``pwd`` and ``bash`` functions, which do not work on Windows. If running on Windows replace
+    it with the path of the working directory.
+
+    If your use case makes use of ``did:nuts`` DIDs, you also need to map port ``5555``, which is used for gRPC traffic by the Nuts network.
+
+You can test whether your Nuts Node is running properly by visiting ``http://localhost:8081/status/diagnostics``. It should
+display diagnostic information about the state of the node.
+
+User
+****
+
+The default user in the container is ``18081`` that is only part of group ``18081``.
+This is a regular user without root privileges to provide an additional level of security.
+If ``datadir`` config value points to a mounted directory, see the section below how to manage privileges needed by the nuts-node.
+
+Volume mounts
+*************
+
+The default working directory within the container is ``/nuts`` that provides defaults for the various configurable data and config paths used:
```

Comment:
Not sure whether this section (mapping config to directories) adds anything for people who want to use it? The (default) paths should be in the configuration reference, and the options and directory names self-explanatory?

Less = more?